packages/scalars/src/scalars/EmailAddress.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    GraphQLError,
+    GraphQLScalarType,
+    GraphQLScalarTypeConfig,
+    Kind,
+} from 'graphql';
+import { z } from 'zod';
+
+export type ScalarType = {
+    input: string;
+    output: string;
+};
+
+export const typedef = 'scalar EmailAddress';
+
+export const schema = z.string().email();
+
```

Comment:
<img width="296" alt="Screenshot 2024-10-10 at 13 21 03" src="https://github.com/user-attachments/assets/142150b3-a56d-46dd-9e08-eb3721ba9707">

Each scalar should also export a TS type in string form. Maybe `export const type`?

Reply:
added!